### DISCUSSION TITLE
Account suspension

### DISCUSSION THREAD

**Discussion by Ashes-19** [community/community#108040] [Other Feature Feedback, Questions, & Ideas] [answered]
### Body

Hi
My main Github account suspend and I cant login , when I contact with support they gave me a ticket , but from more than 2 weeks until now now reply or any thing , they ignore me 
Ticket ID: 2566737
username:AhmedMohamed040
If anyone can help .

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Ashes-19, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
